src/snakemake/functions/params_extra.py

```diff
@@ -9,8 +9,10 @@ def params_extra(wildcards):
     """
     extra=wildcards['extra']
 
-    paths = glob.glob('../mw*/src/snakemake/tables/extra/protectedunderscores.tsv')
+    #paths = glob.glob('../mw*/src/snakemake/tables/extra/protectedunderscores.tsv')
+    paths = glob.glob('../mw*/src/snakemake/tables/extra/protectedunderscores.tsv') + glob.glob("../mw*/out/mw/*_protectedunderscores.tsv")
```

Comment:
This file 
src/snakemake/tables/extra/scRNA_protectedunderscores.tsv 
does not seems to be loaded by the regex. 
Is it supposed to be used elsewhere? Or can I remove it?

Reply:
You can remove it, I forgot to do it sorry